api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -224,4 +224,10 @@
 								 baseTableName="encounter_diagnosis" baseColumnNames="changed_by"
 								 referencedTableName="users" referencedColumnNames="user_id" />
 	</changeSet>
+	<changeSet id="20180226-TRUNK-5331" author="patrick">
+		<dropNotNullConstraint 
```

Comment:
All our changesets have preconditions. You can talk to Ola for help

Reply:
fixed